src/server/controllers/votes.js

```diff
@@ -69,43 +77,48 @@ const options = {
 const combineWithGraph = (graphData, apiData, matchingKey) => {
   const combined = graphData.map((datum) => {
     const match = apiData.find((d) => d[matchingKey] === datum.id);
+
     if (match) {
       return Object.assign(match.dataValues, datum);
     }
+
     return match;
   });
+
   return combined;
 };
 
 const findTopThree = (array, matchingKey) => {
-  // returns top three values, regardless of duplicates in array
-  // unless there are less than three unique values, then returns all values
-  let results = array.map((item) => parseInt(item[matchingKey]));
-  results = Array.from(new Set(results));
-  return results.sort((itemA, itemB) => itemB - itemA).slice(0, 3);
+  // Returns top three values, regardless of duplicates in array unless there
+  // are less than three unique values, then returns all values
+  return array
```

Comment:
why did we remove the cast to set?  what results will this give in a situation like [3, 3, 2, 2, 1, 1, 1, 0]?

Reply:
Good catch! Thats propably wrong. Haven't tested this logic yet but will move on to the reading vote results soon.